pkg/collectsub/collectsub/input/identifier_strings.go

```diff
@@ -72,5 +79,11 @@ func guessUnknownIdentifierString(s string) (*pb.CollectEntry, error) {
 		}, nil
 	}
 
+	if strings.HasPrefix(s, "pkg:") {
```

Comment:
nit: also can use the helper to parse the PURL but i think this should be okay as well

Reply:
Yup. I'll add that.